README.md

```diff
@@ -1,5 +1,7 @@
 ## Neugram
 
+[![Build Status](https://secure.travis-ci.org/neugram/ng.png)](http://travis-ci.org/neugram/ng)
```

Comment:
The http://travis-ci.org/neugram/ng link could/should use https:// scheme.

Also, it might be better to use the badge in .svg format, because that way it renders better on high DPI screens:

https://travis-ci.org/neugram/ng.svg?branch=master

Reference: https://docs.travis-ci.com/user/status-images/.

Reply:
fixed w/ neugram/ng#24